components/patient-data/impl/src/test/java/org/phenotips/data/internal/controller/ParentalAgeControllerTest.java

```diff
@@ -157,7 +157,7 @@ public void loadMaternalAgeNonZero()
 
         Assert.assertEquals("parentalAge", testData.getName());
         Assert.assertTrue(testData.get(MATERNAL_AGE) == AGE_NON_ZERO);
-        Assert.assertNull(testData.get(PATERNAL_AGE));
+        Assert.assertEquals(AGE_ZERO, testData.get(PATERNAL_AGE));
```

Comment:
I believe this test works, but I don't understand why it used to be a null and now it is a 0. Also, what does this case tests, and what does age 0 means - don't we actually want to return null when age is 0?